### PR TITLE
adding hex logo

### PR DESCRIPTION
**Pull request by choldgraf**
This adds @drammock 's hexagon logo to the `logo` folder!

### CODE REVIEW

**Comment by agramfort** [comment]
can you also put the code to produce it?

**Comment by larsoner** [comment]
It might just be a manually masked version of what we already have code for.

**Comment by choldgraf** [comment]
I didn't generate it with code, so not sure where that code is. Maybe @drammock knows?

**Comment by larsoner** [comment]
https://github.com/mne-tools/mne-python/blob/master/logo/generate_mne_logos.py

**Comment by agramfort** [comment]
we need @drammock's knowledge

**Comment by agramfort** [comment]
sklearn stores it's logos in repo: https://github.com/scikit-learn/scikit-learn/tree/master/doc/logos

I am fine merging this.

FYI I just ordered new stickers :)

**Comment by agramfort** [comment]
let's merge this so we don't loose it.

btw I have new stickers :)
